### PR TITLE
Removed remaining self-update code

### PR DESCRIPTION
**Pull request by Bond-009**
As we aren't currently planning on supporting self-updating.

### CODE REVIEW

File: Emby.Notifications/Notifications.cs [hunk 1]
```diff
@@ -124,10 +117,10 @@ private NotificationOptions GetOptions()
             return _config.GetConfiguration<NotificationOptions>("notifications");
         }
 
-        async void _appHost_HasUpdateAvailableChanged(object sender, EventArgs e)
+        private async void _appHost_HasUpdateAvailableChanged(object sender, EventArgs e)
         {
             // This notification is for users who can't auto-update (aka running as service)
```
Comment (JustAMan) [inline Emby.Notifications/Notifications.cs]: Remove the comment?

**Comment by cvium** [comment]
Not enough. There's also https://github.com/jellyfin/jellyfin/blob/master/MediaBrowser.Api/ScheduledTasks/ScheduledTaskService.cs#L200-L208

**Comment by EraYaN** [comment]
There are also all the localization strings.

File: MediaBrowser.Model/Configuration/BaseApplicationConfiguration.cs [hunk 2]
```diff
@@ -7,20 +7,6 @@ namespace MediaBrowser.Model.Configuration
     /// </summary>
     public class BaseApplicationConfiguration
     {
-        // TODO: @bond Remove?
-        /// <summary>
-        /// Gets or sets a value indicating whether [enable debug level logging].
-        /// </summary>
-        /// <value><c>true</c> if [enable debug level logging]; otherwise, <c>false</c>.</value>
-        public bool EnableDebugLevelLogging { get; set; }
```
Comment (JustAMan) [inline MediaBrowser.Model/Configuration/BaseApplicationConfiguration.cs]: why is this removed?

Comment (Bond-009) [reply MediaBrowser.Model/Configuration/BaseApplicationConfiguration.cs]: Wasn't used since the logger update.

**Comment by JustAMan** [comment]
@Bond-009 you now have conflicts here, please fix

@joshuaboniface I think your concerns about strings removal was addressed, can you re-inspect and approve?

**Comment by cvium** [comment]
@joshuaboniface Please re-review this
